presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometrySerde.java

```diff
@@ -61,6 +62,19 @@ public static Slice serialize(OGCGeometry input)
         return output.slice();
     }
 
+    public static Slice serialize(Envelope envelope)
```

Comment:
@arhimondr `writeGeometry` operates on `OGCGeometry` which can't be used to represent an envelope. It is possible to convert an `Envelope` to `OGCGeometry`, but the conversion loses the type. It is no longer possible to tell whether the final polygon is a bounding box without some non-cheap computations.

Reply:
There is no `OCG Geometry` of type `Envelope`, but you can create `OCGEnvelope` type yourself by overriding `OGCSurface`. It will even improve perfomance a bit.